stdlib/websub/src/main/ballerina/websub/commons.bal

```diff
@@ -639,7 +612,7 @@ public type SubscriptionDetails record {|
 |};
 
 function retrieveSubscriberServiceAnnotations(service serviceType) returns SubscriberServiceConfiguration? {
-    any annotData = reflect:getServiceAnnots(serviceType, moduleName = WEBSUB_MODULE_NAME,
+    any annotData = reflect:getServiceAnnotations(serviceType, moduleName = WEBSUB_MODULE_NAME,
                                              ANN_NAME_WEBSUB_SUBSCRIBER_SERVICE_CONFIG);
```

Comment:
We should be able to replace this with 

```ballerina
function retrieveSubscriberServiceAnnotations(service serviceType) returns SubscriberServiceConfiguration? {
    typedesc serviceTypedesc = typeof serviceType;
    return serviceTypedesc.@SubscriberServiceConfig;
} 
```

Can you please check? We might not need the reflect import too.

Reply:
ATM your suggested code results a null pointer at compiler.codegeneretor level. We will change the code once that issue is fixed